### ISSUE TITLE
RTC does nor connect to server

### ISSUE THREAD

**Issue by ramburas** [microsoft/navcontainerhelper#502]
Hi,

When container is created, TRC file ClientUserSetting.config does not have Server value.
So TRC does not run.

**Comment by ernestasju** [comment]
Hi,

I can confirm.

![image](https://user-images.githubusercontent.com/3514528/59848560-1df82c80-936e-11e9-82b8-26bb131e88d7.png)

![image](https://user-images.githubusercontent.com/3514528/59848397-b641e180-936d-11e9-98b1-8ec6a3035a92.png)

Bug happens using latest navcontainerhelper.

To reproduce:

```powershell
<#
NavContainerHelper is version 0.6.1.5
Host is Microsoft Windows 10 Enterprise - 1903
Docker Client Version is 18.09.2
Docker Server Version is 18.09.2
#>
New-BCContainer -accept_eula -containerName test -includeCSide -doNotExportObjectsToText -updateHosts
```

**Comment by freddydk** [comment]
Found the bug, has been there since the traefik implementation.
Will be fixed in the next version.

**Comment by freddydk** [comment]
Should be fixed in 0.6.2.1 - just released.

**Comment by ernestasju** [comment]
Yes, it's fixed. @ramburas is my colleague so this issue can be closed.
